### PR TITLE
Mise à jour du profil de Xavier GOUCHET

### PR DESCRIPTION
**Pull request by xgouchet**

### CODE REVIEW

File: baggers.js [hunk 1]
```diff
@@ -12788,7 +12788,23 @@ var data = {
             "fr",
             "en"
           ]
-        }
+      },
```
Comment (dadoonet) [inline baggers.js]: Small thing but I think we are missing some indentation here.

Comment (xgouchet) [reply baggers.js]: Fixed

File: baggers.js [hunk 2]
```diff
@@ -12788,7 +12788,23 @@ var data = {
             "fr",
             "en"
           ]
-        }
+        },
+      {
```
Comment (dadoonet) [inline baggers.js]: I meant to fix the indentation for the whole new talk :) Sorry if I was unclear

Comment (xgouchet) [reply baggers.js]: done

**Review by dadoonet — APPROVED** [review]

**Pull request merged**